docs/utils/index.js

```diff
@@ -14,6 +14,61 @@ const stripHTML = (str = '') => str.replace(/<[^>]+>/g, '')
 // Remove any double quotes from a string
 const stripQuotes = (str = '') => str.replace(/"/g, '')
 
+export const parseUrl = value => {
+  let anchor = document.createElement('a')
+  anchor.href = value
+
+  // We need to add the anchor to the document to make sure the
+  // `pathname` is correctly detected in any browser
+  document.body.appendChild(anchor)
```

Comment:
Hmmm I didn't do this in the pagination-nav URL checks... I wonder if we should?  In my tests I didn't find I needed to (using chrome).

Reply:
IE 11...

Reply:
Ah... may need to adjust pagination-nav then to do the append/remove

Reply:
Yeah maybe. Using this in production for a while now without issues.

Reply:
For pagination nav, It is only used when vue-router is not used.